### ISSUE TITLE
new assigned Keyboard shortcuts dont work

### ISSUE THREAD

**Issue by rajeshkumarblr** [eranif/codelite#1978]
Hello,

 I  tried  to assign "Ctrl + /" for "Find References" and it does not work at all.
 Even though I selected yes for "overwrite comment", it does not wok
I use ubuntu 16.04. Thanks.

Regards,
Rajesh

**Comment by Randalphwa** [comment]
The problem appears to occur when there is a keyboard conflict -- the keybinding for the conflict is not removed when the new binding is made. Here are steps that can be used to reproduce the bug:

Go to Keyboard Shortcuts
Type "tab" in the search box
Double-click "Previous Tab"
Set the new shortcut to CTRL+A
Click OK
Change the search term from "tab" to "Select"

Result: Select All still has CTRL+A assigned to it
Expected: Select All should no longer have a key assignment

**Comment by eranif** [comment]
I am not sure which version are you using, but was fixed recently

**Comment by Randalphwa** [comment]
I am running version 12 on Windows 7 (64-bit). I just ran through the repro steps I described just to confirm that the problem occurs exactly as I described it. I also was able to reproduce the problem rajeshkumarblr reported -- again using the filter to locate "Find References", assigning it to CTRL+/ and then filtering for "Comment Line" which still has CTRL+/ assigned to it. I also confirmed that neither CTRL+A or CTRL+/ used the new assignments.
